README.adoc

```diff
@@ -331,6 +331,46 @@ spec:
 ...
 ----
 
+=== LDAP Configuration
+To configure authentication strategy to LDAP add the following configuration to the yaml file. If the auth strategy is LDAP. The application will not start without the basic LDAP settings like ldapHost, ldapPort, ldapBase and ldapBindDN.
+
+----
+auth:
+  strategy: ldap
+  ldap:
```

Comment:
I'm going to say I like how this is - an "ldap" section under "auth". I had a previous comment earlier that asked how we should do this - my vote is the way this is already done (ldap subsection under auth).

Reply:
Thanks